File: Trabajo_9/CPP/Actividades/Actividad_2.cpp
#include <iostream>
using namespace std;

class Calculadora{
public:
    void sumar(int a, int b){
        cout << "Respuesta: " << a + b << endl;
    }

    void sumar(int a, int b, int c){
        cout << "Respuesta: " << a + b + c << endl;
    }

    void sumar(double a, double b){
        cout << "Respuesta: " << a + b << endl;
    }

    void sumar(string a, string b){
        cout << a + " " + b << endl;
    }
};



int main(){
    Calculadora calculadora;
    calculadora.sumar(1,2);
    calculadora.sumar(1.3,2.5);
    calculadora.sumar(4,5,2);
    calculadora.sumar("Hola", "Mundo xd");

    return 0;
}

File: Trabajo_9/JAVA/Actividades/Actividad_2.java
package Trabajo_9.Actividades;

class Calculadora {
    public int sumar(int a, int b) {
        System.out.println("Sumando enteros");
        return a + b;
    }
    public double sumar(double a, double b) {
        System.out.println("Sumando doubles");
        return a + b;
    }
    public int sumar(int a, int b, int c) {
        System.out.println("Sumando enteros");
        return a + b + c;
    }
}
public class Actividad_2 {
    public static void main(String[] args) {
        Calculadora calc = new Calculadora();
        System.out.println(calc.sumar(5, 3)); 
        System.out.println(calc.sumar(2.5, 3.7)); 
        System.out.println(calc.sumar(10, 20,30)); 
    }
}